tutorial/1_make_pseudobulks.ipynb

```diff
@@ -0,0 +1,596 @@
+{
```

Comment:
<p>typo/grammar:</p><p>"In this notebook is Part 1 of a BuDDi..." &gt;&gt;&gt;&gt; "This notebook is Part 1..."</p><p> </p><p>Recommendation: </p><p>-This sentence is key to the notebook methods but hard to follow: In this example we have reference stimulated (female single-cell) data, in this analysis the stimulation is Male:Female maps to CTRL:STIM.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/greenelab/buddi_analysis/pull/2/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>

Reply:
<p>good point, hopefully clearer now: <a href="https://github.com/greenelab/buddi_analysis/pull/2/commits/f8169ac1f8988b2ad60db46624ba9ac272a2047b" target="_blank">https://github.com/greenelab/buddi_analysis/pull/2/commits/f8169ac1f8988b2ad60db46624ba9ac272a2047b</a></p>